base/applications/mscutils/eventvwr/eventvwr.c

```diff
@@ -4128,12 +4244,12 @@ EventLogProperties(HINSTANCE hInstance, HWND hWndParent, PEVENTLOGFILTER EventLo
     psp[1].hInstance   = hInstance;
     psp[1].pszTemplate = MAKEINTRESOURCEW(IDD_GENERAL_PAGE);
     psp[1].pfnDlgProc  = GeneralPageWndProc;
-    psp[0].lParam      = (LPARAM)EventLogFilter->EventLogs[0];
+    psp[1].lParam      = (LPARAM)EventLogFilter->EventLogs[0];
 #endif
 
     /* Create the property sheet */
     ret = PropertySheetW(&psh);
-
+    
```

Comment:
What is the extra indent in the blank line meant for?

Reply:
ok